src/modules/Likes.js

```diff
@@ -0,0 +1,43 @@
+const addLike = async (index) => {
+  const scoreURL = 'https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/hHoxKTRKINp4PN8g78ys/likes';
```

Comment:
- [ ] You have done well by declaring the` api url` as a variable that can be accessible anywhere in your project, kindly avoid code repetition as we have it in line 23 below 👇. Doing this will ensure that you follow DRY principle. 👍